### PR TITLE
 PE-11134 Fix import dashboard slug

### PR DESCRIPTION
**Pull request by knowvishnu**
### SUMMARY
PCV Dashboard Slug is not coming after importing the dashboard for new tenants

### ADDITIONAL INFORMATION

- [x] 🐛 Bugfix (non-breaking change which fixes an issue)
- [ ] Introduces new feature or API
- [ ] Changes UI
- [ ] Includes DB Migration
  - [ ] Migration is atomic, supports rollback & is backwards-compatible
  - [ ] Confirm DB migration upgrade and downgrade tested
  - [ ] Runtime estimates and downtime expectations provided
- [ ] 🧹 Chore (removing redunant files, fixing typos etc.)
- [ ] Removes existing feature or API

## Security Considerations

Are there any other security or data concerns to be aware of?

Please discuss the security implications/considerations relevant to the proposed change.
This may include...
* security-relevant design decisions
* concerns
* important discussions
* implementation-specific guidance and pitfalls
* an outline of threats and risks and how they are being addressed.

**NA**

### CODE REVIEW

File: superset/dashboards/commands/importers/v0.py [hunk 1]
```diff
@@ -150,7 +150,7 @@ def alter_positions(
     slices = copy(dashboard_to_import.slices)
 
     # Clearing the slug to avoid conflicts
-    dashboard_to_import.slug = None
+    # dashboard_to_import.slug = None
```
Comment (KuldeepakSingh) [inline superset/dashboards/commands/importers/v0.py:153]: is this something specific to newer version of superset ?

Also let's get rid of the line altogether , no point in keeping commented out code.

Comment (knowvishnu) [reply superset/dashboards/commands/importers/v0.py:153]: yes its broken in new changes

Comment (KuldeepakSingh) [reply superset/dashboards/commands/importers/v0.py:153]: Got it, Please remove the commented out code.

**Review by mukulsaini — APPROVED** [review]

File: superset/dashboards/commands/importers/v0.py (hunk 1 above)
Comment (knowvishnu) [reply superset/dashboards/commands/importers/v0.py]: https://github.com/apache/superset/pull/10118#discussion_r443140489

**Review by KuldeepakSingh — APPROVED** [review]

**Pull request merged**